hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/MergeTableRegionsProcedure.java

```diff
@@ -135,8 +134,8 @@ private static void checkRegionsToMerge(MasterProcedureEnv env, final RegionInfo
           throw new MergeRegionException(msg);
         }
         if (!force && !ri.isAdjacent(previous) && !ri.isOverlap(previous)) {
-          String msg = "Unable to merge non-adjacent or non-overlapping regions " +
-              previous.getShortNameToLog() + ", " + ri.getShortNameToLog() + " when force=false";
+          String msg = "Unable to merge non-adjacent or non-overlapping regions '" +
+              previous.getShortNameToLog() + "', '" + ri.getShortNameToLog() + "' when force=false";
```

Comment:
nit: we need the ' characters? we don't usually do this?

Reply:
This is a clean cherry-pick of your commit to master, the `'` character was part of the original commit. 🙂  It was not pushed to branch-2.2 at that time I found this as a nice improvement.